src/format.ts

```diff
@@ -66,31 +66,29 @@ const generateTypesSection = (
 
 const generateTablesMarkdown = (
   tables: TableDescription[],
-  typeNames: string[]
+  typeNames: Set<string>
 ) => tables.map((t) => generateTableDescription(t, typeNames))
 
 const generateTypesMarkdown = (
   customTypes: CustomType[],
   compositeTypes: CompositeTypeDescription[],
-  typeNames: string[]
+  typeNames: Set<string>
 ) =>
   [
-    generateCustomTypesMarkdown(customTypes, typeNames),
+    generateCustomTypesMarkdown(customTypes),
     generateCompositeTypesMarkdown(compositeTypes, typeNames),
   ].flat()
 
 const generateCustomTypesMarkdown = (
   customTypes: CustomType[],
-  customTypeNames: string[]
 ) => {
   return customTypes
-    .map((custom) => generateCustomTypeMarkdown(custom, customTypeNames))
+    .map((custom) => generateCustomTypeMarkdown(custom))
     .flat()
 }
 
 const generateCustomTypeMarkdown = (
-  custom: CustomType,
-  customTypeNames: string[]
```

Comment:
`customTypeNames` was not used before this right?

Reply:
It wasn't